tests/console/sysstat.pm

```diff
@@ -72,10 +72,14 @@ sub run {
     validate_script_output "mpstat",     sub { /CPU    %usr   %nice    %sys %iowait    %irq   %soft  %steal  %guest  %gnice   %idle/ };
     validate_script_output "sar -u",     sub { /CPU     %user     %nice   %system   %iowait    %steal     %idle/ };
     validate_script_output "sar -n DEV", sub { /IFACE   rxpck\/s   txpck\/s    rxkB\/s    txkB\/s   rxcmp\/s   txcmp\/s  rxmcst\/s   %ifutil/ };
-    if (is_sle('<=15-SP2') || is_leap('<=15.2')) {
-        validate_script_output "sar -b", sub { /tps      rtps      wtps   bread\/s   bwrtn\/s/ };
-    } else {
+    #from version 12.1.2 iostat supports discard I/O statistics.
+    #numify both version variables
+    my $installed_ver = sprintf("%3d%03d%03d", split('\.', script_output('rpm --qf "%{VERSION}\n" -q sysstat'))) + 0;
+    my $ref_ver       = sprintf("%3d%03d%03d", 12, 1, 2) + 0;
+    if ($installed_ver >= $ref_ver) {
```

Comment:
I'm also completely fine with using the sysstat version for such decision! I only have a hint to do the version compare  bit different:
```suggestion
    use version;
    if (version->parse(script_output('rpm --qf "%{VERSION}\n" -q sysstat')) >= version->parse('12.1.2')) {
```

From here https://build.opensuse.org/package/view_file/devel:openQA/os-autoinst-distri-opensuse-deps/_service:obs_scm:os-autoinst-distri-opensuse-deps.spec?expand=1 I would say the "module" `version` is a common dependency.

Reply:
This is lovely! I will update the code, thanks!

Reply:
* [tw](http://kepler.suse.cz/tests/1387#step/sysstat/138)
* [15-SP3-JeOS](http://kepler.suse.cz/tests/1386#step/sysstat/138)

Reply:
I just checked sar manpage and they say 
```
   S_COLORS
              When this variable is set, display statistics in color on the terminal.  Possible values for this variable are never, always or auto (the latter is the default).

```
So maybe we could run `sar` like
```
(unset S_COLORS && sar -b)
```
to avoid that color codes. or just globally unset that `S_COLORS` variable.

Reply:
or try `S_COLORS=never sar -b`

Reply:
Good point, let me try. Thanks!

Reply:
Yup, looks better
http://kepler.suse.cz/tests/1503/file/serial_terminal.txt